run/config_simple_example.cfg

```
@@ -1,5 +1,5 @@
 [model]
-init_file = init_rect30x30.nc
+init_file = init_rect3030.res.nc
```

Why did you change this? Was it a mistake?

```suggestion
init_file = init_rect30x30.res.nc
```